tensorflow-proto/tensorflow-proto.cabal

```diff
@@ -55,19 +63,20 @@ library
                      , Proto.Tensorflow.Core.Protobuf.TensorBundle
                      , Proto.Tensorflow.Core.Protobuf.TensorflowServer
                      , Proto.Tensorflow.Core.Util.Event
+                     , Proto.Tensorflow.Core.Util.Event_Fields
                      , Proto.Tensorflow.Core.Util.MemmappedFileSystem
                      , Proto.Tensorflow.Core.Util.SavedTensorSlice
                      , Proto.Tensorflow.Core.Util.TestLog
-  build-depends:  proto-lens >= 0.2.2 && < 0.3
-                , proto-lens-protoc >= 0.2.2.1 && < 0.3
-                , proto-lens-protobuf-types == 0.2.*
+  build-depends:  proto-lens >= 0.3.0 && < 0.4
+                , proto-lens-protoc >= 0.3.0 && < 0.4
+                , proto-lens-protobuf-types >= 0.3.0 && < 0.5
```

Comment:
Maybe these should also be == 0.3.*, what do you think @judah, @fkm3?
At the very least should be 0.4.